src/components/legend.ts

```diff
@@ -269,11 +275,12 @@ export module Component {
         });
       });
 
-      entries.select("circle")
-          .attr("cx", layout.textHeight / 2)
-          .attr("cy", layout.textHeight / 2)
-          .attr("r",  layout.textHeight * 0.3)
-          .attr("fill", (value: string) => this._scale.scale(value) );
+      entries.select("path").attr("d", this.symbolGenerator())
+                            .attr("transform", "translate(" + (layout.textHeight / 2) + "," + layout.textHeight / 2 + ") " +
+                                               "scale(" + (layout.textHeight * 0.3 / 50) + ")")
```

Comment:
Can we try to cut down on the magic numbers? I know the 0.3 was from before, but we should save off the 50 somewhere, probably in `SymbolGenerators`